cmd/examples/example1.go

```diff
@@ -48,19 +48,32 @@ func main() {
 	}
 
 	// Pass the Filter struct to the FilteredInstanceTypes function of your
-	// selector instance to get a list of filtered instance types and their details
+	// selector instance to get a list of filtered instance types and their details.
 	instanceTypesSlice, err := instanceSelector.FilterInstanceTypes(filters)
 	if err != nil {
 		fmt.Printf("Oh no, there was an error getting instance types: %v", err)
 		return
 	}
 
-	// Pass in your list of instance type details to the appropriate output function
-	// in order to format the instance types as printable strings.
-	maxResults := 100
+	// Pass in the list of instance type details to the SortInstanceTypes function
+	// if you wish to sort the instances based on set filters.
+	sortFilter := "instance-type-name"
+	sortDirection := "ascending"
+	instanceTypesSlice, err = instanceSelector.SortInstanceTypes(instanceTypesSlice, &sortFilter, &sortDirection)
+	if err != nil {
+		fmt.Printf("Oh no, there was an error sorting instance types: %v", err)
+		return
+	}
+
+	// Truncate results and format them for output with your desired formatting function.
+	// All formatting functions can be found here:
+	// https://github.com/aws/amazon-ec2-instance-selector/blob/main/pkg/selector/outputs/outputs.go
+	// Examples of formatted outputs can be found here:
+	// https://github.com/aws/amazon-ec2-instance-selector#examples
+	maxResults := 10
 	instanceTypesSlice, _, err = outputs.TruncateResults(&maxResults, instanceTypesSlice)
```

Comment:
I don't think we really need this for outputs. It's trivial to truncate a slice. It's not clear why this returns 3 values too.  What error could be returned from a truncation?

Reply:
For this type of func, I'd drop the `error` and do something reasonable.

For example, 

```
TruncateResults(-1, 5results) => [], 5
TruncateResults(0, 5results) => [], 5
TruncateResults(10, 5results) => [1,2,3,4,5], 0
```